index.js

```
@@ -180,6 +183,19 @@ Hook.prototype.initialize = function initialize () {
   this.status = this.status.stdout.toString().trim()
   this.root = this.root.stdout.toString().trim()
 
+  if (fs.existsSync(path.join(this.root, '.pre-commit.json'))) {
+    let preCommitConfig
+
+    try {
+      const raw = fs.readFileSync(path.join(this.root, '.pre-commit.json'), 'utf-8').toString()
+      preCommitConfig = JSON.parse(raw)
+    } catch (err) {}
```

this should differentiate between file not found and other permission errors, e.g. if the file exists and we cannot access it.

---

Do you mean that we should log the error object to the console?, should we exit the process with an error code too or should we ignore the error and continue?.

---

to me we should `warn` the user in a case of a present but not readable file; if there is no file we load the default configuration

---

I'm fine with either option. Silently ignoring it is not good.